swat/cas/table.py

```diff
@@ -126,7 +126,7 @@ def _get_unique(seq, lowercase=False):
     Parameters
     ----------
     lowercase : boolean
-        Should values be compared in a case-insensitive way?
+        If true, compare elements in a case-insensitive way
```

Comment:
This 'true' should be capitalized to match the other changes.  After this change, I'm fine with the whole set of commits.

Reply:
Sounds good. I just updated it. Thanks!